docs/docs/guides/integrate/identity-providers/_ldap_warn.mdx

```diff
@@ -0,0 +1,4 @@
+:::caution
+LDAP does send your user passwords in clear text. Due to this we do not recommend using it with the ZITADEL cloud.
```

Comment:
This doesn't apply to local LDAP, right?
Because it's imported there.

Reply:
you should not use a local ldap with zitadel cloud

Reply:
to show the user that this should not be done